cdap-ui/app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.js

```diff
@@ -100,39 +112,21 @@ export function enableSchedule(
   MyScheduleApi.get(scheduleParams).subscribe(
     () => {
       // Schedule exist, update it
-      MyScheduleApi.update(scheduleParams, requestObj).subscribe(
-        () => {
-          MyScheduleApi.enableTrigger(scheduleParams).subscribe(
-            () => {
-              // fetch list triggers
-              fetchTriggersAndApps(activePipeline, workflowName);
-            },
-            () => {}
-          );
-        },
-        (err) => {
-          console.log('error update', err);
-        }
-      );
+      MyScheduleApi.update(scheduleParams, requestObj).subscribe(() => {
```

Comment:
May not be in this PR but can we chain this with `flatMap` instead of nesting inside `subscribe`? This way if update fails we can fallback to create and the rest of the steps (`enablingTrigger` and `fetchTriggersAndApps`are the same)